packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -122,11 +123,15 @@ export async function fetchSnapshotWithRedeem(
 	snapshotDownloader: (
 		finalOdspResolvedUrl: IOdspResolvedUrl,
 		storageToken: string,
+		fetchFullSnapshot: boolean,
+		loadingGroupIds: string[] | undefined,
 		snapshotOptions: ISnapshotOptions | undefined,
 		controller?: AbortController,
 	) => Promise<ISnapshotRequestAndResponseOptions>,
 	putInCache: (valueWithEpoch: IVersionedValueWithEpoch) => Promise<void>,
 	removeEntries: () => Promise<void>,
+	fetchFullSnapshot: boolean,
+	loadingGroupIds: string[] | undefined,
```

Comment:
Can we have something like that?
```ts
type GroupIdInfo = string[] | undefined;
isFetchingFullSnapshot(info: GroupIdInfo): boolean;
```
and ensure that
1. All layers have exactly same semantics (with exception the last layer that does fetch() maybe) on how to interpret GroupIdInfo (i.e. isFetchingFullSnapshot() implementation would be correct across all layers)
2. Not pass fetchFullSnapshot boolean through APIs?

Reply:
That said, see remark below on "full snapshot" vs. "snapshot with default group". Not sure, maybe better say "default snapshot"? Does not sound well either. Maybe "initial snapshot"? Or "non-filtered snapshot" (though that's a lie, as it's filtered to include only default groups)

Reply:
That said, please start with this comment: https://github.com/microsoft/FluidFramework/pull/19593/files#r1491351998

Reply:
Resolving as per the offline discussion. 
For type, I don't see need for creating a separate type, and we now handle it using utility apis and comments.